test/FunctionalTests/XunitLogger.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// See License.txt in the project root for license information
+
+using System;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Xunit.Abstractions;
+
+namespace EntropyTests
+{
+    public class XunitLogger : ILogger, IDisposable
```

Comment:
Still helps us a lot because the tests that deploy a server produce lots of console output. It's better to have this captured by xunit since console.writeline interleaves different tests.